recipes/ignition-cmake/all/conanfile.py

```diff
@@ -49,15 +52,22 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        version_major = tools.Version(self.version).major
```

Comment:
why need to remove it now?

Reply:
the files that are being removed are the library provided config.cmake file which is aligned with the recommendations of CCI in [this link](https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-are-cmake-findconfig-files-and-pkg-config-files-not-packaged)

Reply:
For find-xxx.cmake and xxx-config.cmake, YES, you should use the generated by the package manager. However, I'm not confident removing the utilities files, it not a regular one which we usually found on other projects. Doing a quick read, it provides `ignition-cmake2::ignotion-cmake2-utilities` target, which is not listed on cpp_info (consider it a bug on that recipe).